src/components/data-management/UploadDetailsModal.jsx

```diff
@@ -119,8 +118,6 @@ const UploadDetailsModal = (props) => {
           )}
         {renderFields(extraFields)}
 
-        {!isNotUploadedModal && renderFields({ Filename: name })}
```

Comment:
This was unnecessary. Category and Filename showed the same information so I just kept Category